src/test/java/org/opensrp/repository/postgres/ClientsRepositoryTest.java

```diff
@@ -512,7 +514,37 @@ public void shouldGetMemberCountHouseholdHeadProviderByClients() {
 		id.add("28caef27-d1b3-497b-8a55-95Rf2f0f6e24");
 		List<HouseholdClient> householdClient = clientsRepository.selectMemberCountHouseholdHeadProviderByClients("", id,
 		    "ec_family");
-		assertEquals(householdClient.size(), 0);
 		assertNotNull(householdClient);
+		assertEquals(0, householdClient.size());
```

Comment:
This assert should expect a non-zero value. 
This will make sure that 
1.  We capture the correct functionality based on the title of this test **_assertNotNull(householdClient)_** 
2.  Catch a regression bug where correct number of records are returned after future code updates